src/components/common/NameRequestInfo.vue

```diff
@@ -109,7 +84,7 @@
           <v-col
             cols="12"
             sm="3"
-            class="pr-4"
+            class="pt-1 pr-4"
```

Comment:
Before:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/b5356460-9c85-49ad-bef7-8de2306e592d)

After:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/54c2b47d-8be9-49f1-9dc7-d6480583af8d)

Ditto for line 133 below.